docs/upstream-integrations/build-a-miniapp.md

```diff
@@ -0,0 +1,304 @@
+---
+id: build-a-miniapp
+title: Build a miniapp with Collab.Land Actions
+sidebar_position: 3
+---
+
+Collab.Land Actions is a set of tools for building and testing custom actions for Discord interactions. With Collab.Land Actions, the setup and configuration of the Collab.Land infrastructure, including Docker, AWS, authentications and securities, are handled for you by default.
+
+To get started, use the `hello-action` [template provided on GitHub](https://github.com/abridged/collabland-hello-action/). This template provides a basic structure for building custom actions on top of the Collab.Land engine. Once your custom action is built, it can be run on a TestFlight mini-app within the Collab.Land QA bot to execute the business logic you've developed.
+
+After testing, you can then install the action on your Discord server and test your implementation end to end without any oversight or dependency on Collab.Land. This allows for efficient contribution and extension to Collab.Land without the overhead of manual setup.
+
```

Comment:
Wish we had a logic flow diagram on here like we do in `build-a-custom-action.md`

Reply:
The logic flow in the custom action docs applies here as well. They don't need it here since this is a tutorial style doc